BlackNectar/BlackNectar/StoresDataStruct.swift

```diff
@@ -11,63 +11,65 @@ import CoreLocation
 import Foundation
 import UIKit
 
-// Creating data structure for JSON Request
-struct StoresInfo {
+// Creates data structure for JSON Request
+struct Stores {
```

Comment:
Nice! Well done. Except it should be `Store`, in the singular.
Remember, we declare `Int`, `Double`, etc, and not `Ints`, `Doubles`. That would be an array.